test/unit/gossip-server-statefulset.bats

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bats
+
+load _helpers
+
+@test "global/gossipEncryption: disabled in server StatefulSet by default" {
```

Comment:
As with the client test, it would be good to check that the `-encrypted` flag does not get set in this case.

Reply:
Fixed in 87c49d849b8231ff5d628f518b2b20a44d19246e.